src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -302,7 +302,18 @@ public static TestPackage MakeTestPackage(ConsoleOptions options)
             if (options.Verbose)
                 package.AddSetting("Verbose", true);
 
+            if (options.DebugTests)
+            {
+                package.AddSetting(PackageSettings.DebugTests, true);
+
+                if (options.NumWorkers < 0)
+                    package.AddSetting(PackageSettings.NumberOfTestWorkers, 0);
```

Comment:
When workers is 1, we still use a queue. There are actually three workers created, the single one specified, another for tests marked as non-parallel and a third for tests in the STA.

When it's set to 0, we do none of that and revert to the simpler approach, without queueing, that we used for NUnit V2.

We may want to unify this code in the future and eliminate workers=0, but for now it's there. If we do that, I woujld want to put in some optimizations for workers=1.